backend/app/services/llm_service.py

```
@@ -128,10 +123,160 @@ def _extract_json_from_response(self, response: str) -> Dict[str, Any]:
+
+    def _persist_feedback(self, domain: str) -> None:
+        """
+        Persist feedback data for a specific domain to the configuration.
+
+        Args:
+            domain: The domain whose feedback data should be persisted.
+        """
+        try:
+            feedback_list = list(self.feedback_data[domain])
+            self.config_service.save_feedback_data(domain, feedback_list)
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)